wgpu-core/src/device/resource.rs

```diff
@@ -2926,6 +2916,48 @@ impl Device {
                     stride: vb_state.array_stride,
                 });
             }
+            let mut last_stride = 0;
+            for attribute in vb_state.attributes.iter() {
+                if attribute.offset % attribute.format.size().min(4) != 0 {
```

Comment:
```suggestion
                let required_offset_alignment = attribute.format.size().min(4);
                if (attribute.offset % required_offset_alignment) != 0 {
```

just a bit clearer, there's a lot going on here